fix for: error: more than one instance of overloaded function "abs" matches the argument list
author: Kenneth Hoste (HPC-UGent)
--- flann-1.8.4-src/src/cpp/flann/algorithms/kdtree_index.h.orig	2017-11-22 20:15:58.975950087 +0100
+++ flann-1.8.4-src/src/cpp/flann/algorithms/kdtree_index.h	2017-11-22 20:16:24.166478426 +0100
@@ -36,6 +36,7 @@
 #include <cassert>
 #include <cstring>
 #include <stdarg.h>
+#include <cmath>
 
 #include "flann/general.h"
 #include "flann/algorithms/nn_index.h"
@@ -663,7 +664,7 @@
             ElementType max_span = 0;
             size_t div_feat = 0;
             for (size_t i=0;i<veclen_;++i) {
-                ElementType span = abs(point[i]-leaf_point[i]);
+                ElementType span = std::abs(point[i]-leaf_point[i]);
                 if (span > max_span) {
                     max_span = span;
                     div_feat = i;
